portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/clustering/clusteringframework/ClusteringCache.testcase

```diff
@@ -330,4 +267,119 @@ definition {
 			scriptFile = "groovy-script-portal-cache-get.groovy");
 	}
 
+	@priority = 4
+	test SessionReplicatedIdAndAttributesAreTheSameInBothNodes {
+		property cluster.session.replicated = "true";
+		property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}portlet.session.replicate.enabled=true";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		Clustering.deployClusteringModule(nodePort = 8080);
+
+		Clustering.deployClusteringModule(nodePort = 9080);
+
+		ProductMenu.gotoPortlet(
+			category = "Site Builder",
+			portlet = "Pages");
+
+		PagesAdmin.addPage(pageName = "Session Replication Test");
+
+		Navigator.gotoPage(pageName = "Session Replication Test");
+
+		Portlet.addPG(portletName = "ClusterSampleSessionReplication");
+
+		Open(value1 = "http://localhost:8080/web/guest/session-replication-test");
+
+		var sampleData = selenium.getText("ClusterSamplePortlet#STORED_DATA_VALUE");
+		var sessionId = selenium.getText("ClusterSamplePortlet#SESSION_ID");
+
+		Open(value1 = "http://localhost:9080/web/guest/session-replication-test");
+
+		AssertTextEquals(
+			locator1 = "ClusterSamplePortlet#STORED_DATA_VALUE",
+			value1 = ${sampleData});
+
+		AssertTextEquals(
+			locator1 = "ClusterSamplePortlet#SESSION_ID",
+			value1 = ${sessionId});
+
+		JSONUser.addUser(
+			userEmailAddress = "test1@liferay.com",
+			userFirstName = "Test1",
+			userLastName = "Test1",
+			userScreenName = "Test1");
+
+		JSONUser.setFirstPassword(
+			agreeToTermsAndAnswerReminderQuery = "true",
+			requireReset = "false",
+			userEmailAddress = "test1@liferay.com");
+
+		User.firstLoginUI(
+			password = PropsUtil.get("default.admin.password"),
+			specificURL = "http://www.able.com:9080",
+			userEmailAddress = "test1@liferay.com");
+
+		Open(value1 = "http://www.able.com:9080/web/guest/session-replication-test");
+
+		Refresh();
```

Comment:
Note that this refresh() is still necessary

Reply:
@julschong Why?

Reply:
Why you do not need refresh for the default user?